packages/app/components/Queue/TA/TAQueueListItem.tsx

```diff
@@ -19,6 +21,22 @@ const Container = styled.div<{ selected: boolean }>`
   align-items: flex-start;
 
   padding-top: 16px;
+  return string;
+}
+
+const Container = styled.div<{ selected: boolean }>\`
```

Comment:
On my machine it's not liking the `\` before the back tick. Also this `Container` is overriding the one on line 19

```suggestion
const Container = styled.div<{ selected: boolean }>`
```

Reply:
without it the following is not properly evaluated as a string. It is weird idk if we need that section